src/index.html

```diff
@@ -9,10 +9,51 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="styles/style.css"
     />
```

Comment:
Keep your attributes correctly formatted. Since the <link> element has more than two attributes, each attribute should start on a new line with proper indentation.